.changeset/hot-vans-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/theme-check-common': minor
```

Comment:
I would call this a patch but maybe renaming the check makes it a minor? We probably want to explicitly call out the rename.